fiftyone/utils/cvat.py

```diff
@@ -222,21 +240,82 @@ def import_annotations(
         anno_backend.delete_run(sample_collection, anno_key)
 
 
-def _get_task_filepaths(api, task_id, data_map, ignored_filenames):
+def _get_task_filepaths(
+    api,
+    task_id,
+    data_map,
+    ignored_filenames,
+    data_dir=None,
+    download_media=False,
+):
     resp = api.get(api.task_data_meta_url(task_id)).json()
+    start_frame = resp.get("start_frame", None)
+    stop_frame = resp.get("stop_frame", None)
+    chunk_size = resp.get("chunk_size", None)
     filenames = [frame["name"] for frame in resp["frames"]]
 
     filepaths = []
-    for filename in filenames:
+    for frame_id, filename in enumerate(filenames):
```

Comment:
Collecting the download tasks and running them via a thread pool would help with efficiency here:

```py
def _download_media(tasks, num_workers):
    logger.info("Downloading media...")
    if not num_workers or num_workers <= 1:
        with fou.ProgressBar() as pb:
            for task in pb(tasks):
                _do_download_media(task)
    else:
        with multiprocessing.dummy.Pool(processes=num_workers) as pool:
            with fou.ProgressBar(total=len(tasks)) as pb:
                results = pool.imap_unordered(_do_download_media, tasks)
                for _ in pb(results):
                    pass
```